### PR TITLE
Clean up

### PR DESCRIPTION
**Pull request by mlpollar**

### CODE REVIEW

**Comment by AJJiro** [comment]
Nice changes in frontend/src/index.css from 40-44. 
Nice color change in rontend/src/index.js in line 13.

Having a color picker is nice when we do annotation so people make different notes.
No changes needed.

File: frontend/src/index.css [hunk 1]
```diff
@@ -26,23 +26,60 @@ span {
...
+  max-height: 100%;*/
+}
+
+.save { 
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  /*width: 1400px;
+  max-height: 100%;*/
+}
```
Comment (Shegab21) [inline frontend/src/index.css:82]: nice organization on all the formatting

File: frontend/src/index.js [hunk 2]
```diff
@@ -37,40 +37,50 @@ class InitialState extends Component {
...
             >
-             
-            <img src={Save} alt ="Save Button"/>
+              <img src={Save} height='40' alt ="Save Button"/>
 
             </button>
+
             <button
-              style={{height: '230px', width : '240px'}}
+              style={{height: '50px', width : '50px'}}
```
Comment (Shegab21) [inline frontend/src/index.js:67]: i like the smaller sizes

File: frontend/src/index.js [hunk 3]
```diff
@@ -136,40 +144,30 @@ class InitialState extends Component {
...
-            lazyRadius={this.state.lazyRadius}
-            canvasWidth={this.state.width}
-            canvasHeight={this.state.height}
-            
-          />
+          
           <p>
-            The following is a disabled canvas with a hidden grid that we use to
-            load & show your saved drawing.
+            See you're saved template below!
```
Comment (Shegab21) [inline frontend/src/index.js]: *your

File: frontend/src/index.js [hunk 4]
```diff
@@ -136,40 +144,30 @@ class InitialState extends Component {
...
+          <div class="save" className={classNames.tools}>
+            <button
+              style= {{order: 2, height: '100px', width: '300px'}}
+              onClick={() => {
+                this.loadableCanvas.loadSaveData(
+                  localStorage.getItem("savedDrawing")
+                );
+              }}
+            >
+              CLICK ME! To see your saved changes.
```
Comment (Shegab21) [inline frontend/src/index.js:161]: maybe "to load" your saved changes?

File: frontend/src/index.js [hunk 5]
```diff
@@ -37,40 +37,50 @@ class InitialState extends Component {
...
+              brushColor={this.state.color}
+              brushRadius={this.state.brushRadius}
+              lazyRadius={this.state.lazyRadius}
+              canvasWidth={this.state.width}
+              canvasHeight={this.state.height}
+              
+            />
             <button
-              style={{height: '230px', width : '240px'}}
+              style={{height: '50px', width : '50px'}}
```
Comment (Shegab21) [inline frontend/src/index.js:54]: maybe set these dimensions elsewhere as a constant?

**Review by Shegab21 — APPROVED** [review]

**Pull request merged**